docs/topics/auth/default.txt

```diff
@@ -1006,6 +1019,19 @@ implementation details see :ref:`using-the-views`.
       authenticated users accessing the login page will be redirected as if
       they had just successfully logged in. Defaults to ``False``.
 
+      .. warning::
+
+          If you use this feature attackers might be able to retrieve the
```

Comment:
I failed to move the inline link into a separate definition. Got an idea how / where to move that?